swift-generator/src/main/resources/templates/java/common.st

```diff
@@ -399,3 +414,7 @@ public void <field.javaSetterName>(final <field.javaType> <field.javaName>) {
     this.name = "<field.name>";
 }
 >>
+
+generatedComment(context) ::= <<
+// Generated by swift-generator from <context.sourceIDLPath> on <context.currentTime>
```

Comment:
The path should start with repository root, it should not include full path (such as /home/sazonovk/...)